build.sbt

```diff
@@ -6,6 +6,8 @@ ThisBuild / tlBaseVersion := "1.5"
 ThisBuild / crossScalaVersions := Seq(Scala213, Scala212, Scala3)
 ThisBuild / scalaVersion := Scala213
 ThisBuild / startYear := Some(2018)
+ThisBuild / tlSitePublishBranch := None
```

Comment:
```suggestion
```

If you publish the site for 1.x tags, then it will override the 2.x site. Either we should set up versioned documentation, or you should never published the site from the 1.x branch/tags ever. WDYT?

Reply:
Eh, you're probably right. I think sbt-typelevel should have some explicit settings to disable publishing site. It could be odd from some angle but it makes sense in the case of the multibranch project without a versioned site. I think it's worth discussing it in the sbt-typelevel space.

Reply:
@armanbilge I've started https://github.com/typelevel/sbt-typelevel/issues/137, so would be lovely to get your thoughts on it.